utils/parsing/config.yaml

```diff
@@ -31,7 +31,7 @@ partitioning:
   metadata_include: []
   partition_endpoint: 'http://localhost'
   unstructured_port: 8005
-  partition_by_api: True
```

Comment:
@snova-jorgep This does actually need to be true but we need to change 
-  'set to true if using Docker container' to
- 'set' to true if using API server.

We are still parsing via API server, it's just that it's a local one vs docker

Reply:
fixed